x-pack/plugins/actions/server/plugin.ts

```diff
@@ -108,6 +109,9 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
   }
 
   public async setup(core: CoreSetup, plugins: ActionsPluginsSetup): Promise<PluginSetupContract> {
+    this.isESOUsingEphemeralEncryptionKey =
+      plugins.encryptedSavedObjects.usingEphemeralEncryptionKey;
+
```

Comment:
It seems like we should log an **error** message here, when `isESOUsingEphemeralEncryptionKey` is true.  Will help folks looking at error logs caused by the exceptions we're now throwing.  Something like the following, stolen from one of the other messages in this PR:

> Unable to initialize the alerting plugin due to the Encrypted Saved Objects plugin using an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in kibana.yml.

Reply:
Good point, I went ahead with the message below. The changes are within this commit https://github.com/elastic/kibana/pull/56906/commits/1cf1caceb4abc2478305bbaff0234dc9211a77b3 and I went ahead and made the same for the alerting plugin 👍 

> APIs are disabled due to the Encrypted Saved Objects plugin using an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in kibana.yml.